addons/vehicles/functions/fnc_startEngine.sqf

```diff
@@ -20,6 +20,8 @@ params ["_vehicle", "_isEngineOn"];
 
 if (!_isEngineOn || {floor abs speed _vehicle > 0 || {!isNull isVehicleCargo _vehicle}}) exitWith {};
 
+private _startupDelay = getNumber (configOf _vehicle >> QGVAR(engineStartDelay));
```

Comment:
```suggestion
private _startupDelay = _vehicle getVariable [QGVAR(engineStartDelay), -1];
if (_startupDelay == -1) then {
    _startupDelay = getNumber (configOf _vehicle >> QGVAR(engineStartDelay));
};
```
This should satisfy @BrettMayson's request.

Reply:
Check new commit, please. Maybe I should revert and apply your's

Reply:
What you have looks good. Only thing I would maybe name the public function `setVehicleStartDelay`, as you are setting it, not adding it.

Reply:
Done